index.html

```diff
@@ -889,67 +889,133 @@ <h3>Supporting algorithms</h3>
+          </li>
+          <li>
+            If |topLevelBC| does not have [=top-level traversable/system focus=], return false.
+          </li>
+          <li>
+            Let |focusedDocument| be the |topLevelBC|'s
+            <a href="https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context">
+              currently focused area</a>'s [=Node/node document=].
+          </li>
+          <li>
+            If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with
+            |focusedDocument|'s [=origin=], return true.
+          </li>
+          <li>
+            Otherwise, return false.
+          </li>
+        </ol>
+      </p>
+      <p>
+        The <dfn>passes privacy test</dfn> steps given the argument |observer:PressureObserver| and
+        its [=relevant global object=] |relevantGlobal|, are as follows:
+        <ol>
+          <li>
+            If |relevantGlobal| is a {{WorkerGlobalScope}} object:
+            <ol>
+              <li>
+                If |relevantGlobal|'s relevant worker is not a <a href="https://html.spec.whatwg.org/multipage/workers.html#active-needed-worker">
+                active needed worker</a>, return false.
+              </li>
+              <li>
+                If |relevantGlobal| is a {{DedicatedWorkerGlobalScope}} object:
+                <ol>
+                  <li>
+                    Let |ownerGlobal| be |relevantGlobal|'s [=owning global object=].
+                  </li>
+                  <li>
+                    Assert: |ownerGlobal| is not null.
+                  </li>
+                  <li>
+                    Return the result of running [=passes window privacy test=] with |observer| and |ownerGlobal|.
+                  </li>
+                </ol>
+              </li>
+              <li>
+                If |relevantGlobal| is a {{SharedWorkerGlobalScope}} object:
+                <ol>
+                  <li>
+                    [=list/For each=] |owner| in |relevantGlobal|'s [=WorkerGlobalScope/owner set=]:
+                    <ol>
+                      <li>
+                        Assert: |owner| is {{Document}}.
+                      </li>
+                      <li>
+                        If the result of running [=passes window privacy test=] with |observer| and |owner|'s
+                        [=relevant global object=] is true, return true, or else [=iteration/continue=].
+                      </li>
+                    </ol>
+                    <li>
+                      Return false.
+                    </li>
+                  </li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+          <li>
+            If |relevantGlobal| is not a {{Window}} object, return false.
+          </li>
+          <li>
+            Return the result of running [=passes window privacy test=] with |observer| and |relevantGlobal|.
```

Comment:
Personally I find it clearer if you invert the last 2 steps so that the algorithm reads like this:

1. If relevantGlobal is worker scope, do this
2. If relevantGlobal is window, do this
3. Return false